kratos/python_scripts/print_info_in_file_process.py

```diff
@@ -0,0 +1,139 @@
+            "help"                     : "This process prints nodal/elemental information ina .txt file",
+            "mesh_id"                  : 0,
+            "model_part_name"          : "please_specify_model_part_name",
+            "variable_name"            : "SPECIFY_VARIABLE_NAME",
+            "results_type"             : "not_provided_by_default",
+            "file_name"                : "info_file.txt",
+            "output_control_type"      : "step",
+            "integration_point_number" : 0,
+            "erase_previous_info"      : true,
+            "output_interval"          : 1,
+            "sum_results_from_multiple_entities" : false
+        }""")
+        settings.ValidateAndAssignDefaults(default_settings)
+
+        self.variable = KratosMultiphysics.KratosGlobals.GetVariable(settings["variable_name"].GetString())
+        self.results_type = settings["results_type"].GetString()
+        if not self.results_type == "nodal_historical" and not self.results_type == "nodal_non_historical" and not self.results_type == "elemental":
+            raise NameError("results_type not correct, must be nodal_historical, nodal_non_historical or elemental")
+        self.is_nodal_results_type = "nodal" in self.results_type
+        self.file_name = settings["file_name"].GetString()
+        self.output_control_type = settings["output_control_type"].GetString()
+        self.output_interval = settings["output_interval"].GetDouble()
+        self.model_part = Model[settings["model_part_name"].GetString()]
+        self.instant_previous_plot = 0.0
+        self.integration_point = settings["integration_point_number"].GetInt()
+        self.sum_results_from_multiple_entites = settings["sum_results_from_multiple_entites"].GetBool()
+
+        is_mpi_execution = (self.model_part.GetCommunicator().TotalProcesses() > 1)
+        if is_mpi_execution:
+            raise RuntimeError('MPI not supported yet')
+
+        if not self.sum_results_from_multiple_entites:
+            if self.is_nodal_results_type and self.model_part.NumberOfNodes() > 1:
+                raise NameError("The sum_results_from_multiple_entites is false but more than one node is given...")
+            if not self.is_nodal_results_type and self.model_part.NumberOfElements() > 1:
+                raise NameError("The sum_results_from_multiple_entites is false but more than one element is given...")
+
+        open_file_aproach = "a"
+        if settings["erase_previous_info"].GetBool():
+            open_file_aproach = "w"
+
+        with open(self.file_name, open_file_aproach) as plot_file:
+            plot_file = open(self.file_name, open_file_aproach)
+            plot_file.write("# In this file we print the " + settings["results_type"].GetString() + " " + settings["variable_name"].GetString() + " in the ModelPart: " + settings["model_part_name"].GetString() + "\n\n")
+            plot_file.write("# TIME\t\t" + settings["variable_name"].GetString() + "\n")
```

Comment:
please use the [`TimeBasedAsciiFileWriterUtility`](https://github.com/KratosMultiphysics/Kratos/blob/master/kratos/python_scripts/time_based_ascii_file_writer_utility.py)

=> this way many features like restart and proper folder creation are supported natively